autoload/lsp/lsp.vim

```diff
@@ -1019,16 +1019,18 @@ export def TextDocFormat(range_args: number, line1: number, line2: number)
     return
   endif
 
-  var lspserver: dict<any> = buf.CurbufGetServerChecked('documentFormatting')
+  silent var lspserver: dict<any> = buf.CurbufGetServerChecked('documentFormatting')
```

Comment:
Using `:silent` will suppress all errors, suggest using `buf.BufLspServerGet(bufnr(), 'documentFormatting')` without `:silent` instead

Reply:
It was necessary to hide the warning message from `CurbufGetServerChecked()` when the feature is not supported